BackEnd/src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -10,11 +10,18 @@
 
 
 public interface DashboardRepository extends JpaRepository<Event, Integer> {
-        @Query(
-        value = "SELECT * FROM Event e WHERE (e.start_Date >= :startDate AND e.end_Date <= :endDate) AND e.employee_id = :employeeId",
-        nativeQuery=true
-        )
+    @Query(
+            value = "SELECT * FROM Event e WHERE (e.start_Date >= :startDate AND e.end_Date <= :endDate) AND e.employee_id = :employeeId",
+            nativeQuery = true
+    )
     List<Event> findCalendarMonthEventsForEmployee(@Param("employeeId") int employeeId,
                                                    @Param("startDate") LocalDate startDate,
-                                                   @Param("endDate") LocalDate endDate );
+                                                   @Param("endDate") LocalDate endDate);
+
+    @Query(value = "SELECT * FROM Event e INNER JOIN Contract c on e.employee_id = c.employee_id where (e.start_date >= :startDate AND e.end_Date <= :endDate)" +
+            "AND c.team_id in(select team_id FROM Contract d where d.employee_id = :employeeId and c.end_date IS NULL)",
```

Comment:
This needs to select contracts that have the start date before now and the end date null or adfter now.